client/src/AddFriend.js

```diff
@@ -46,4 +87,16 @@ class AddFriend extends React.Component {
   }
 }
 
+// takes in data from axios.get and outputs array of friends
+function getfriends(res) {
+  const friends = res.data.rows
+  let flist = []
+  for (let i = 0; i < friends.length; i++) {
+    const fname = friends[i].friendfname.charAt(0).toUpperCase() + friends[i].friendfname.slice(1)
+    const lname = friends[i].friendlname.charAt(0).toUpperCase() + friends[i].friendlname.slice(1)
+    flist[i] = fname + " " + lname
+  }
+  return flist
```

Comment:
This kind of heavy formatting should be done before sending data to the client (at the express API level). See `routes/index`